src/mon/MgrMonitor.cc

```diff
@@ -960,7 +960,7 @@ bool MgrMonitor::preprocess_command(MonOpRequestRef op)
     f->dump_string("active_name", map.get_active_name());
     f->dump_unsigned("num_standby", map.get_num_standby());
     f->close_section();
-    f->flush(rdata)
+    f->flush(rdata);
```

Comment:
could you squash this change into 808ace6c975bb4db362f043c82ef9d44c243d1e0 whose title is "mon: add 'mgr stat' command"?